CHANGES.md

```diff
@@ -1,3 +1,30 @@
+## edge-19.12.2
+
+* CLI
+  * Added support for injecting CronJobs and ReplicaSets, and the ability to use
+    them as targets in the CLI subcomands
+  * Added ability to update the trust anchor and issuer certificates during
+    `linkerd upgrade` with the new flags `--identity-issuer-certificate-file`,
+    `--identity-issuer-key-file` and `identity-trust-anchors-file`
+
+* Controller
+  * Fixed bug causing pods having security context capabilities to fail to be
```

Comment:
would it make more sense to change "having" to "with" ? TIOLI

Reply:
Maybe reword it: "Fixed inject failures for pods with security context capabilities"